zdm/survey.py

```diff
@@ -713,44 +970,160 @@ def load_survey(survey_name:str, state:parameters.State, dmvals:np.ndarray,
     """
     print(f"Loading survey: {survey_name}")
     if sdir is None:
-        sdir = os.path.join(resource_filename('zdm', 'data'), 'Surveys')
+        sdir = os.path.join(
+            resource_filename('zdm', 'data'), 'Surveys')
+        if original:
+            sdir = os.path.join(sdir, 'Original')
 
     # Hard code real surveys
     if survey_name == 'CRAFT/FE':
-        dfile = 'CRAFT_class_I_and_II.dat'
-        Nbeams = 5
+        dfile = 'CRAFT_class_I_and_II'
+        nbins = 5
     elif survey_name == 'CRAFT/ICS':
-        dfile = 'CRAFT_ICS.dat'
-        Nbeams = 5
+        dfile = 'CRAFT_ICS'
+        nbins = 5
     elif survey_name == 'CRAFT/ICS892':
-        dfile = 'CRAFT_ICS_892.dat'
-        Nbeams = 5
+        dfile = 'CRAFT_ICS_892'
+        nbins = 5
     elif survey_name == 'CRAFT/ICS1632':
-        dfile = 'CRAFT_ICS_1632.dat'
-        Nbeams = 5
+        dfile = 'CRAFT_ICS_1632'
+        nbins = 5
     elif survey_name == 'PKS/Mb':
-        dfile = 'parkes_mb_class_I_and_II.dat'
-        Nbeams = 10
+        dfile = 'parkes_mb_class_I_and_II'
+        nbins = 10
     elif 'private' in survey_name: 
-        dfile = survey_name+'.dat'
-        if Nbeams is None:
-            raise IOError("You must specify Nbeams with a private survey file")
+        dfile = survey_name
+        if nbins is None:
+            raise IOError("You must specify nbins with a private survey file")
     else: # Should only be used for MC analysis
-        dfile = survey_name+'.dat'
-        if Nbeams is None:
-            Nbeams = 5
+        dfile = survey_name
+        if nbins is None:
+            nbins = 5
         else:
-            Nbeams=Nbeams
+            nbins=nbins
+
+    if original:
+        dfile += '.dat'
+    else:
+        dfile += '.ecsv'
 
     # Do it
-    srvy=Survey()
-    srvy.name = survey_name
-    srvy.process_survey_file(os.path.join(sdir, dfile), NFRB=NFRB, iFRB=iFRB)
-    #srvy.process_survey_file(os.path.join(sdir, dfile), NFRB=NFRB, iFRB=iFRB)
-    srvy.init_DMEG(state.MW.DMhalo)
-    srvy.init_beam(nbins=Nbeams, method=state.beam.Bmethod, plot=False,
-                thresh=state.beam.Bthresh) # tells the survey to use the beam file
-    pwidths,pprobs=make_widths(srvy,state)
-    _ = srvy.get_efficiency_from_wlist(dmvals,pwidths,pprobs)
+    if original:
```

Comment:
Thanks for keeping original functionality. At some point I expect we can drop backwards compatibility, but great to have it for now.